Dockerfile

```diff
@@ -25,7 +25,6 @@ RUN useradd -m vagrant  -u $IMAGE_USERID -p vagrant
 
 USER vagrant
 WORKDIR /vagrant
+
 # Setup bundler
 RUN bundle config build.nokogiri --use-system-libraries
-
-CMD ["bundle", "exec", "rails", "server"] 
```

Comment:
Is this needed, to remove this CMD line? If so I would comment it in the commit message.

Reply:
The problem is that the first `vagrant up` can't start it because the gems are not installed. I've tried moving `bundle install` in the Dockerfile, but that did not really work well.

On consecutive `vagrant up`s the provisioning is skipped and `bundle exec rails server` is not executed. Maybe the better alternative is to enable ssh which should allow vagrant to provision again and automatically start rails (right now the developer has to `docker attach software_web` and `bundle exec rails server` every time which is not really nice.

Reply:
I would try using docker-compose instead docker from vagrant. But I'm not going to stop this from being merged. So fine for me.